src/components/Retrospective/RetrospectiveReportSubComponent.js

```diff
@@ -0,0 +1,234 @@
+import React, { useEffect, useReducer } from 'react';
+import {
+  TextField,
+  TableContainer,
+  Paper,
+  TableHead,
+  TableCell,
+  TableBody,
+  TableRow,
+  Table,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import '../../App.css';
+
+const useStyles = makeStyles({
+  formControl: {
+    minWidth: '100%',
+  },
+  inputLabel: {
+    color: '#bbb',
+  },
+  userInput: {
+    color: 'white',
+  },
+  modal: {
+    maxWidth: '400px',
+    minWidth: '400px',
+    margin: 'auto',
+    marginTop: '30px',
+    marginBottom: '30px',
+    overflow: 'auto',
+  },
+});
+
+const RetrospectiveReport = ({
+  userEmail,
+  saveChangesConnection,
+  refreshUIConnection,
+}) => {
+  const classes = useStyles();
+
+  const initialState = {
+    tableData: null,
+    tasksModified: null,
+  };
+  const [state, setState] = useReducer(
+    (state, newState) => ({ ...state, ...newState }),
+    initialState
+  );
+
+  useEffect(() => {
+    //Connect the child to the parent
+    saveChangesConnection(saveChangesChildConnection);
+    refreshUIConnection(refreshUI);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const refreshUI = () => {
+    let sprint = JSON.parse(sessionStorage.getItem('sprint'));
```

Comment:
this is nitpicky and doesn't need to be fixed but in the future, try to use const for absolutely everything in JS if you're not reassigning it